src/cascadia/TerminalControl/ControlInteractivity.cpp

```diff
@@ -182,7 +182,12 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         }
         else if (_canSendVTMouseInput(modifiers))
         {
-            _core->SendMouseEvent(terminalPosition, pointerUpdateKind, modifiers, 0, buttonState);
+            const auto adjustment = _core->ScrollOffset() > 0 ? _core->BufferHeight() - _core->ScrollOffset() - _core->ViewHeight() : 0;

```

Comment:
I'm surprised we don't already have a thing that calculates "how far back from the top line are we scrolled"... hmm.